script.js

```diff
@@ -1,4 +1,36 @@
+//There is player 1
+//There is player 2,
+//Diceroll code
+//There is an addition of numbers
+//
+
+var gamemode = "Yellow";
+var player1 = [0, 0];
+var player2 = [0, 0];
```

Comment:
If you are attempting to define an array here, there is no need to put in values within the array. E.g., var player1 = [] would suffice! :)